scripts/image

```diff
@@ -48,7 +48,11 @@ if [ "$OFFICIAL" = "yes" ]; then
   LIBREELEC_BUILD="official"
 else
   if [ -n "$BUILDER_NAME" ]; then
-    LIBREELEC_BUILD="$BUILDER_NAME"
+    if [ "$BUILDER_NAME" = "official" ]; then
```

Comment:
Logic is wrong here - LIBREELEC_BUILD will be left unset when BUILDER is "official".

Final nit-pick:
```
if [ "${BUILDER_NAME,,}" = "official" ]; then
```
just in case anyone tried using `Official` or some other variation.

This should fix both issues:
```
if [ "$OFFICIAL" = "yes" ]; then
  LIBREELEC_BUILD="official"
else
  [ "${BUILDER_NAME,,}" = "official" ] && BUILDER=
  if [ -n "$BUILDER_NAME" ]; then
    LIBREELEC_BUILD="$BUILDER_NAME"
  else
    LIBREELEC_BUILD="community"
  fi
fi
```